src/controlCenter/widgets/mpris/mpris.vala

```diff
@@ -99,6 +100,15 @@ namespace SwayNotificationCenter.Widgets.Mpris {
                 bool blur_found;
                 bool? blur = get_prop<bool> (config, "blur", out blur_found);
                 if (blur_found) mpris_config.blur = blur;
+
+                Json.Array ? blacklist = get_prop_array (config, "blacklist");
+                if (blacklist != null) {
+                    string[] apps = new string[blacklist.get_length ()];
+                    for (int i = 0; i < blacklist.get_length (); i++) {
+                        apps[i] = blacklist.get_string_element (i);
```

Comment:
Nit: Should double-check that it's a string

Reply:
done